src/main.rs

```diff
@@ -100,7 +100,7 @@ fn main() {
                 .takes_value(true)
                 .possible_values(&compile_vals)
                 .default_value("envoy")
-                .help("Sets what to compile to:  the simulator (sim) or envoy wasm filter (rs)"),
+                .help("Sets what to compile to: the simulator (sim) or envoy wasm filter (rs)"),
```

Comment:
We should change this from rs to something like "e" because they're both in rust so the abbreviation doesn't make a lot of sense.

Reply:
It is already envoy, I updated the description.